porting/introduction/Intro.rst

```diff
@@ -0,0 +1,134 @@
+
+The next component of Ubuntu Touch is a pre-compiled root filesystem which needs to be installed on the device. This component does not communicate directly with the device hardware. Instead, this communication is mediated by a Hardware Abstraction Layer (HAL) which needs to be built for each specific device, because each device has its specific hardware architecture. This component is called Halium  and is available in different versions (5.1 which is largely obsolete, 7.1, 9 and 10 as of writing) corresponding to different Android versions.
+
+The `Halium project <https://halium.org/>`_ enables Linux systems to run on Android hardware. It is a joint effort by multiple mobile operating systems, notably Plasma mobile, Lune OS and UBports. 
+
+UBports porting builds on top of Halium porting. Consequently, you will be using both the Halium porting guide and the UBports porting guide. At times it may also be helpful to test with one of the other operating systems to debug a problem from different angles.
+
+Halium is an indispensible part of an Ubuntu Touch port and is available in the form of open source software. Developing a new version of Halium is a very considerable task which is why only a few versions of Halium are available. Each port of Ubuntu Touch has to be based on one of the available Halium versions and vendor blobs from the corresponding Android version. See the first two columns of the table below for details.
+
+Thus an Ubuntu Touch port is composed of the these components:
+    * The Ubuntu Touch (UT) root filesystem (rootfs)
+    * `Halium <https://halium.org/>`_ (contained in the boot and system images)
+    * The vendor blobs
+
+You, the porter, need to build Halium (in part or in whole, depending on :ref:`porting method <Porting-methods>`) and install this together with the Ubuntu Touch rootfs in order to create a functioning Ubuntu Touch port. 
+
+.. _Android-and-Halium-versions:
+
+Android and Halium versions
+---------------------------
+
+Halium is built using source code for a modified version of the Android operating system called LineageOS (see `the LineagesOS website <https://lineageos.org/>`_ and `wiki <https://wiki.lineageos.org/>`_). The required source code is available online and needs to be downloaded and configured to build the correct Halium version for each individual device port. The table below shows which versions are required for the different Halium versions.
+
+===============  ==============  ================
+Android version  Halium version  Lineage OS (LOS)
+===============  ==============  ================
+7.1              7.1             14.1            
+9.0              9.0             16.0            
+10.0             10.0            17.0
+===============  ==============  ================
+
+.. _What-is-a-GSI:
+
+What is a GSI?
```

Comment:
the title is a little off. nobody is asking this question. nobody has mentioned GSI yet.